trivup/apps/ZookeeperApp.py

```diff
@@ -46,7 +46,8 @@ def __init__(self, cluster, conf=None, on=None, bin_path=None):
         Exposes 'address' (host:port) for other apps.
         """
         super(ZookeeperApp, self).__init__(cluster, conf=conf, on=on)
-        self.conf['port'] = trivup.TcpPortAllocator(self.cluster).next(self)
+        self.conf['port'] = trivup.TcpPortAllocator(self.cluster).next(self, 
```

Comment:
Please add zk_port to the Config: list in the doc-string above.

Reply:
Indeed, I forgot. Will fix that.